lib/chef/knife/opc_org_user_remove.rb

```diff
@@ -22,8 +22,15 @@ class OpcOrgUserRemove < Chef::Knife
     banner "knife opc org user remove ORG_NAME USER_NAME"
     attr_accessor :org_name, :username
 
+    option :force_remove_from_admins,
+    :long => '--force_remove_from_admins',
```

Comment:
I'd prefer -'s rather than _'s.  We could also perhaps just change this to `--force`?